src/QuickMarcEditor/QuickMarcEditor.js

```diff
@@ -57,10 +66,18 @@ const QuickMarcEditor = ({ instance, onClose, handleSubmit, initialValues, submi
               data-test-quick-marc-editor={instance?.id}
               data-testid="quick-marc-editor"
             >
-              <QuickMarcEditorRows
-                fields={records}
-                name="records"
-              />
+              <FormSpy
+                subscription={{ values: true }}
+              >
+                {() => (
+                  <QuickMarcEditorRows
```

Comment:
how many times QuickMarcEditorRows is rerendered with `FormSpy`? 

imo it shouldn't render anything, smth like that

```
<form>
<FormSpy>
```